diff --git a/src/Composer/Compiler.php b/src/Composer/Compiler.php
index 6fccb8e..383c502 100644
--- a/src/Composer/Compiler.php
+++ b/src/Composer/Compiler.php
@@ -43,32 +43,9 @@ public function compile($pharFile = 'composer.phar')
             unlink($pharFile);
         }
 
-        $process = new Process('git log --pretty="%H" -n1 HEAD', __DIR__);
-        if ($process->run() != 0) {
-            throw new \RuntimeException('Can\'t run git log. You must ensure to run compile from composer git repository clone and that git binary is available.');
-        }
-        $this->version = trim($process->getOutput());
-
-        $process = new Process('git log -n1 --pretty=%ci HEAD', __DIR__);
-        if ($process->run() != 0) {
-            throw new \RuntimeException('Can\'t run git log. You must ensure to run compile from composer git repository clone and that git binary is available.');
-        }
-
-        $this->versionDate = new \DateTime(trim($process->getOutput()));
+        $this->versionDate = new \DateTime('2017-12-18 13:09:18 +0000');
         $this->versionDate->setTimezone(new \DateTimeZone('UTC'));
-
-        $process = new Process('git describe --tags --exact-match HEAD');
-        if ($process->run() == 0) {
-            $this->version = trim($process->getOutput());
-        } else {
-            // get branch-alias defined in composer.json for dev-master (if any)
-            $localConfig = __DIR__.'/../../composer.json';
-            $file = new JsonFile($localConfig);
-            $localConfig = $file->read();
-            if (isset($localConfig['extra']['branch-alias']['dev-master'])) {
-                $this->branchAliasVersion = $localConfig['extra']['branch-alias']['dev-master'];
-            }
-        }
+        $this->version = '1.5.6';
 
         $phar = new \Phar($pharFile, 0, 'composer.phar');
         $phar->setSignatureAlgorithm(\Phar::SHA1);
